ReactSkia/core_modules/RSkInputEventManager.cpp

```diff
@@ -121,12 +121,20 @@ void RSkInputEventManager::processKey(RskKeyInput &keyInput) {
       return;//don't propagate key further
     }
   }
+  
 #if defined(TARGET_OS_TV) && TARGET_OS_TV
   sendNotificationWithEventType(
       RNSKeyMap[keyInput.key_],
       currentFocused ? currentFocused->getComponentData().tag : -1,
       keyInput.action_, nullptr);
 #endif //TARGET_OS_TV
+  /*Sending Events to the registered callback*/
+  eventCallbackMutex_.lock();
+  for (auto pair : eventCallbackMap_){
+    auto clientCallback = pair.second;
+    clientCallback(keyInput);
+  }
+  eventCallbackMutex_.unlock();
```

Comment:
I feel Spatial navigation should be given more priority so that navigation can be faster.